news.html

```diff
@@ -1,44 +1,49 @@
-<span>
-    <h4>
-        <a target="_blank" rel="noopener noreferrer"
-           href="https://medium.com/dockstore/dockstore-achieves-the-best-available-security-standard-in-the-industry-ce194b017089">
-        Dockstore achieves best available security standard</a>
-    </h4>
-    <span>Learn more about our journey to NIST-800-53 compliance in our blog post.</span>
-</span>
-
-<span>
-    <h4>
-        <a target="_blank" rel="noopener noreferrer"
-           href="https://www.youtube.com/watch?v=GWBg8gJDpiw">
-        Latest webinar from Dockstore</a>
-    </h4>
-    <span>Watch our latest webinar introducting Dockstore to the eLwazi Consortium.</span>
-</span>
-
-<span>
-    <h4>
-        <a target="_blank" rel="noopener noreferrer"
-           href="https://medium.com/dockstore/out-with-the-old-in-with-the-tools-1ad78132822d">
-        Dockstore releases a simplified tool definition in 1.12</a>
-    </h4>
-    <span>Read more about changes in 1.12 in our blog post.</span>
-</span>
-
-<span>
-    <h4>
-        <a target="_blank" rel="noopener noreferrer"
-           href="https://github.com/dockstore/dockstore/releases/tag/1.12.0">
-        1.12 release</a>
-    </h4>
-    <span>Get more information on our 1.12.x series releases here.</span>
-</span>
-
-<span>
-    <h4>
-        <a target="_blank" rel="noopener noreferrer"
-           href="https://medium.com/dockstore/dockstore-partners-with-aws-agc-to-make-launching-workflows-quick-and-easy-7213510dabd8">
-        Launch Dockstore workflows with AWS Genomics CLI</a>
-    </h4>
-    <span>Learn about our new partnership with Amazon Web Services Genomics CLI.</span>
-</span>
+<div fxLayout="column" fxLayoutAlign="space-between" class="h-100">
+  <div fxLayout fxLayout.lt-sm="row wrap" fxLayoutAlign="space-between end">
+    <span>
+      <h5 class="mb-0">
```

Comment:
Not necessary to change, because I think it works in [almost?] all the browsers, but fwiw, this is illegal HTML, see https://ucsc-cgl.atlassian.net/browse/DOCK-2116

Reply:
TIL ^ Going to change them to `div`s now since I don't see this file being updated in the near future...